.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: rust
+rust:
+- stable
+
```

Comment:
* build in nightly
* build in no_std
* build with feature cached-block-device
* build with feature **cached-block-device in no_std** (the SunriseOS case)
* clippy rule